templates/rhel8.saphana.tpl.yaml

```diff
@@ -153,42 +153,55 @@ objects:
             kubernetes.io/hostname: ${TARGET_NODE_NAME}
 parameters:
   - description: Name for the new VM
+    displayName: Name
     name: NAME
     required: true
   - description: Amount of memory
+    displayName: Memory
     name: MEMORY
     value: 24Gi
   - description: Amount of memory overhead for qemu
+    displayName: Memory overhead
     name: MEMORY_OVERHEAD
     value: 44Gi
   - description: Amount of cores
+    displayName: CPU Cores
     name: CPU_CORES
     value: "4"
   - description: Amount of threads
+    displayName: CPU threads
     name: CPU_THREADS
     value: "1"
   - description: Amount of sockets
+    displayName: CPU sockets
     name: CPU_SOCKETS
     value: "1"
   - description: Name of the sriov network1
+    displayName: sriov network1
```

Comment:
nitpick suggestion: when adding a network interface on the web UI modal,
the naming is `SR-IOV` and not `sriov` maybe it's best to change for consistancy